communication/marketing-team/storytelling-resources/social-guidelines.md

```diff
@@ -4,7 +4,7 @@
 
 ## General guidelines
 
-All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](./CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
+All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
```

Comment:
```suggestion
All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
```
For inter-repo documentation, a relative link is preferred 👍 
(they're not impacted by branch renames etc)